api/dfs.js

```diff
@@ -244,11 +249,15 @@ async function fetchDeployComponentsMetadata(accountId, projectId) {
  * @param {string} projectName
  * @returns {Promise}
  */
-async function provisionBuild(accountId, projectName) {
+async function provisionBuild(accountId, projectName, platformVersion = '') {
+  const platformQueryString = platformVersion
+    ? `?platformVersion=${platformVersion}`
```

Comment:
I imagine we can do something similar here. If there is no `platformVersion` we are likely better off excluding it from the request rather than passing it as an empty string

Reply:
Addressed in https://github.com/HubSpot/cli-lib/pull/12/commits/760274c2778a1f3b3f3d9ee3a815e217ee24c0e4